sources/model/GGRealTimeMontior.m

```diff
@@ -550,7 +558,7 @@ - (BOOL)handleSocketIODidReceiveEvent:(nonnull NSString *)eventName withData:(no
         
         id existingDelegate = [self delegateForWaypointID:wpid];
         
-        NSLog(@"delegate: %@ should udpate waypoint %@ ETA to: %@ withData:%@", existingDelegate, wpid, eta , eventData);
+        NSLog(@"delegate: %@ should update waypoint %@ ETA to: %@ withData:%@", existingDelegate, wpid, eta , eventData);
         
         if (existingDelegate) {
             [existingDelegate waypointDidUpdatedWaypointId:wpid eta:etaToDate];
```

Comment:
also here respondsToSelector?

Reply:
This method is not optional, it is required.